README.md

```diff
@@ -3,44 +3,23 @@
 # Electronic Cats PN7150
 Library/Driver for NXP PN7150 NFC device
 
-Optimized for fast design-in, NXP’s PN7150 NFC solutions fully comply with the NFC Forum, including Linux®, AndroidTM, and WinIoT drivers and support NCI 1.0 host communication.
+Optimized for fast design-in, NXP's PN7150 NFC solutions fully comply with the NFC Forum, including Linux®, AndroidTM, and WinIoT drivers, and support NCI 1.0 host communication.
 
-PN7150 is the high-performance version of PN7120, the plug’n play NFC solution for easy integration into any OS environment, reducing Bill of Material (BOM) size and cost. PN71xx controllers are ideal for home-automation applications such as gateways and work seamlessly with NFC connected tags.
+PN7150 is the high-performance version of PN7120, the plug'n-play NFC solution for easy integration into any OS environment, reducing Bill of Material (BOM) size and cost. PN71xx controllers are ideal for home automation applications such as gateways and work seamlessly with NFC-connected tags.
 
+## Quick Installing
 
-### Quick Installing
+To install, use the [Arduino Library Manager](https://support.arduino.cc/hc/en-us/articles/5145457742236-Add-libraries-to-Arduino-IDE) and search for "PN7150" and install the PN7150 by Electronic Cats library.
 
-To install, use the Arduino Library Manager and search for "PN7150" and install the library.
+##  Wiki and Getting Started
+For more information please visit: [**Getting Started in our Wiki**](https://github.com/ElectronicCats/ElectronicCats-PN7150/wiki)
 
-### Manual Installing
-To install this library:
-
- - Install it using the Arduino Library manager ("Sketch" -> "Include
-   Library" -> "Manage Libraries..."), or
- - Download a zipfile from github using the "Download ZIP" button and
-   install it using the IDE ("Sketch" -> "Include Library" -> "Add .ZIP
-   Library..."
- - Clone this git repository into your sketchbook/libraries folder.
- For more info, see https://www.arduino.cc/en/Guide/Libraries
-
-### Compatibility
```

Comment:
Let's keep this list, take a look to: https://github.com/ElectronicCats/mpu6050/blob/master/README.md